sdk/appconfiguration/Azure.Data.AppConfiguration/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.3.0-beta.4 (Unreleased)
+## 1.3.0 (Unreleased)
```

Comment:
If we are preparing for release, can we update the changelog?

Reply:
Updated it! I haven't included a release date yet since it may depend on architect review.